envbuilder.go

```diff
@@ -104,7 +105,10 @@ func Run(ctx context.Context, opts options.Options, preExec ...func()) error {
 		return fmt.Errorf("set uid: %w", err)
 	}
 
-	opts.Logger(log.LevelInfo, "=== Running init command as user %q: %q", args.UserInfo.user.Username, append([]string{opts.InitCommand}, args.InitArgs...))
+	initCmd := strings.Join(append([]string{opts.InitCommand}, args.InitArgs...), " ")
```

Comment:
An unfortunate side-effect of this is that it’s lossy. We’re printing `”/bin/bash -c #!…` which is actually not true without quotes, so you can’t copy paste the output and run it like you could previously (as an example).

Reply:
The idea came from a customer call. An alternative option is to soft wrap in the UI but I figured it would also be cheap enough to do on the envbuilder end. I didn't think about the copy-pastability of the output!